exec/csnex/Disassembly.cs

```diff
@@ -0,0 +1,788 @@
+                case Opcode.LES: LES(); break;
+                case Opcode.GES: GES(); break;
+                case Opcode.EQY: EQY(); break;
+                case Opcode.NEY: NEY(); break;
+                case Opcode.LTY: LTY(); break;
+                case Opcode.GTY: GTY(); break;
+                case Opcode.LEY: LEY(); break;
+                case Opcode.GEY: GEY(); break;
+                case Opcode.EQA: EQA(); break;
+                case Opcode.NEA: NEA(); break;
+                case Opcode.EQD: EQD(); break;
+                case Opcode.NED: NED(); break;
+                case Opcode.EQP: EQP(); break;
+                case Opcode.NEP: NEP(); break;
+                case Opcode.EQV: EQV(); break;
+                case Opcode.NEV: NEV(); break;
+                case Opcode.ANDB: ANDB(); break;
+                case Opcode.ORB: ORB(); break;
+                case Opcode.NOTB: NOTB(); break;
+                case Opcode.INDEXAR: INDEXAR(); break;
+                case Opcode.INDEXAW: INDEXAW(); break;
+                case Opcode.INDEXAV: INDEXAV(); break;
+                case Opcode.INDEXAN: INDEXAN(); break;
+                case Opcode.INDEXDR: INDEXDR(); break;
+                case Opcode.INDEXDW: INDEXDW(); break;
+                case Opcode.INDEXDV: INDEXDV(); break;
+                case Opcode.INA: INA(); break;
+                case Opcode.IND: IND(); break;
+                case Opcode.CALLP: CALLP(); break;
+                case Opcode.CALLF: CALLF(); break;
+                case Opcode.CALLMF: CALLMF(); break;
+                case Opcode.CALLI: CALLI(); break;
+                case Opcode.JUMP: JUMP(); break;
+                case Opcode.JF: JF(); break;
+                case Opcode.JT: JT(); break;
+                case Opcode.DUP: DUP(); break;
+                case Opcode.DUPX1: DUPX1(); break;
+                case Opcode.DROP: DROP(); break;
+                case Opcode.RET: RET(); break;
+                case Opcode.CONSA: CONSA(); break;
+                case Opcode.CONSD: CONSD(); break;
+                case Opcode.EXCEPT: EXCEPT(); break;
+                case Opcode.ALLOC: ALLOC(); break;
+                case Opcode.PUSHNIL: PUSHNIL(); break;
+                case Opcode.RESETC: RESETC(); break;
+                case Opcode.PUSHPEG: PUSHPEG(); break;
+                case Opcode.JUMPTBL: JUMPTBL(); break;
+                case Opcode.CALLX: CALLX(); break;
+                case Opcode.SWAP: SWAP(); break;
+                case Opcode.DROPN: DROPN(); break;
+                case Opcode.PUSHFP: PUSHFP(); break;
+                case Opcode.CALLV: CALLV(); break;
+                case Opcode.PUSHCI: PUSHCI(); break;
+                default:
+                    stream.AppendFormat("Unknown opcode: {0}", bytecode.code[index]);
+                    index++;
+                    break;
+            }
+        }
+
+        public string DisassembleInstruction(Bytecode obj, int ip)
```

Comment:
There's still no reason for this function to be a member function. Make it a `static` function, and construct a new `InstructionDisassembler` here to do the actual work.